src/frontend/screens/ObservationEdit/ThumbnailAndActionTab.tsx

```diff
@@ -108,6 +112,7 @@ export const ThumbnailAndActionTab: FC<ThumbnailAndActionTab> = ({
         closeSheet={closeAudioPermissionSheet}
         isOpen={isAudioPermissionSheetOpen}
         sheetRef={audioPermissionSheetRef}
+        onPermissionGranted={handleAudioPermissionGranted}
```

Comment:
nit: i would lean towards not specifying this for now instead of having a no-op placeholder, especially since we don't have an immediate plan to address anytime soon.